components/tabcordion/src/Tab.js

```diff
@@ -135,7 +137,7 @@ export const Tab = forwardRef(
 		useEffect(() => {
 			if (mode === 'tabs') {
 				if (selected) {
-					onOpen(tabId);
+					onOpen({ idx, tabId });
 				} else {
 					onClose(tabId);
```

Comment:
We should pass the idx and tabId to onClose, onOpening and onClosing to be consistent